server/app.coffee

```diff
@@ -1,2 +1,8 @@
 Meteor.startup ->
-    
+    config =
+        run_seed: process.env.SEED == "true"
+
+    if config.run_seed
+        Meteor.call 'seedPosts'
+        Meteor.call 'seedTopics'
+        Meteor.call 'seedSections'
```

Comment:
A to ciekawe... Czyli w sumie nie sprawdziłeś, czy działa, nie? W weekend ogarnę deployment i role